src/address-watch/agent.js

```diff
@@ -23,6 +10,49 @@ function createAlert(agentState, address, contractName, type, severity) {
   });
 }
 
+const validateConfig = (config) => {
+  let ok = false;
+  let errMsg = "";
+
+  if (config["developerAbbreviation"] === undefined) {
+      errMsg = `No developerAbbreviation found`;
+      return { ok, errMsg };
+  }
+  if (config["protocolName"] === undefined) {
+      errMsg = `No protocolName found`;
+      return { ok, errMsg };
+  }
+  if (config["protocolAbbreviation"] === undefined) {
+      errMsg = `No protocolAbbreviation found`;
+      return { ok, errMsg };
+  }
+  if (config["contracts"] === undefined) {
+      errMsg = `No contracts found`;
+      return { ok, errMsg };
+  }
+  if (Object.keys(config.contracts).length === 0) {
+    errMsg = 'Must supply at least one address to watch';
+    return { ok, errMsg };
+  }
```

Comment:
It looks like there is a mismatch between the example configuration file and what this function is validating:
```json
			"agentType": "address-watch",
			"name": "agent_2",
			"addressList": {
				"0xFe1A6056EE03235f30f7a48407A5673BBf25eD48": {
					"name": "AaveDeployer",
					"type": "Info",
					"severity": "Info"
				}
			}
```

Also, the entries in the Object should be validated as well (i.e. contract address, alert type, and alert severity).  I don't know if the `name` field is used in the bot, but if it is, it should be validated as existing and not being an empty string.